packages/notebook/src/widget.ts

```diff
@@ -3261,6 +3300,29 @@ export namespace Notebook {
      */
     export interface IOptions extends StaticNotebook.ContentFactory.IOptions {}
   }
+
+  /**
+   * An interface for the paste cell event.
+   */
+  export interface IPasteCells {
```

Comment:
Came back to merge it, but here is a nitpick: should it be

```suggestion
  export interface IPastedCells {
```

?

Reply:
I updated it.